apps/pay/components/layouts/username-layout.tsx

```diff
@@ -55,123 +24,16 @@ const UsernameLayoutContainer = ({ children, username }: Props) => {
   }
 
   return (
-    <div className={`${openSideBar && styles.container_bg} ${styles.container}`}>
+    <div className={styles.main_container}>
```

Comment:
```suggestion
    <div className={styles.mainContainer}>
```

let's use camelCase name for consistency

Reply:
This is used throughout the project in CSS. Maybe topic for another PR?